src/SFA.DAS.ApplyService.Data/GatewayRepository.cs

```diff
@@ -168,6 +169,7 @@ public async Task<List<RoatpGatewaySummaryItem>> GetInProgressGatewayApplication
                             JSON_VALUE(apply.ApplyData, '$.GatewayReviewDetails.ClarificationRequestedOn') AS ClarificationRequestedDate
 	                      FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
+                          LEFT OUTER JOIN FinancialReview fr on fr.ApplicationId = apply.ApplicationId
```

Comment:
I wonder if we can save doing the left outer join here as we know PMO is after Gateway